rust/core/Cargo.toml

```
@@ -16,31 +16,41 @@
 # under the License.
 
 [package]
-name = "adbc_core"
+authors.workspace = true
+categories.workspace = true
 description = "Public abstract API, driver manager and driver exporter"
-version.workspace = true
+documentation.workspace = true
 edition.workspace = true
-rust-version.workspace = true
-authors.workspace = true
+homepage.workspace = true
+keywords.workspace = true
 license.workspace = true
+name = "adbc_core"
 readme = "../README.md"
-documentation.workspace = true
-homepage.workspace = true
 repository.workspace = true
-keywords.workspace = true
-categories.workspace = true
+rust-version.workspace = true
+version.workspace = true
 
 [features]
 default = []
-driver_manager = ["dep:libloading"]
+driver_manager = ["dep:toml", "dep:libloading", "dep:windows-sys", "dep:windows-registry"]
+driver_manager_test_lib = ["driver_manager"]
+driver_manager_test_manifest_user = ["driver_manager_test_lib"]
 
 [dependencies]
 arrow-array.workspace = true
 arrow-schema.workspace = true
-libloading = { version = "0.8", optional = true }
+libloading = {version = "0.8", optional = true}
+toml = {version = "0.8.23", default-features = false, features = [
+    "parse", "display",
+], optional = true}
```

0.9 was released yesterday: https://github.com/toml-rs/toml/blob/main/crates/toml/CHANGELOG.md#090---2025-07-08

```suggestion
toml = { version = "0.9.0", default-features = false, features = [
    "parse", "display", "std",
], optional = true }
```